src/bundle.js

```diff
@@ -0,0 +1,45 @@
+import './assets/stylesheets/style.scss';
+import { creator, createModal } from './modules/helpers';
+import { filterData, getWeather } from './modules/data';
+import { createWeatherDisplay, setValues, symbols } from './modules/display';
+import { formSearch, units } from './modules/form';
+import { createBGContainer, setBGImage, iconList } from './modules/background';
+
+const content = document.getElementById('content');
+const main = creator(content, 'main', 'append');
+formSearch(main, units);
+createBGContainer(main);
+createWeatherDisplay(main);
+
+const options = {
+  enableHighAccuracy: true,
+  timeout: 5000,
+  maximumAge: 0,
+};
+
+function manageResponse(weatherData) {
+  if (weatherData) {
+    const allData = filterData(weatherData);
+    setBGImage(iconList, allData[6]);
+    setValues(symbols, ...allData);
+  } else {
+    const modalContent = createModal(main);
+    modalContent.innerHTML = 'The city provided could not be found.';
+    document.getElementById('error-msg').style.display = 'block';
+  }
+}
+
+async function success(pos) {
+  const crd = pos.coords;
+  const weatherData = await getWeather(crd);
+  manageResponse(weatherData);
+}
+
+async function error(err) {
+  const weatherData = await getWeather('Tokyo+&units=metric');
+  manageResponse(weatherData);
+  return err;
+}
```

Comment:
**Let's make use of the arrow function instead of using the `function` keyword**
![image](https://user-images.githubusercontent.com/31897434/92687892-3b7b3280-f334-11ea-860a-cf5e33ef18f8.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/